specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -9393,7 +9393,7 @@
         },
         "hyperVGeneration": {
           "$ref": "#/definitions/HyperVGenerationType",
-          "description": "Specifies the HyperVGenerationType of the VirtualMachine created from the image. This propery might be required if not available from the blob or disk resource."
+          "description": "Specifies the HyperVGenerationType of the VirtualMachine created from the image. If the image source is a blob, then we need the user to specify the value. If the source is managed resource like disk or snapshot, the user may need to specify the property if we can't deduce it from the source managed resource. See  https://docs.microsoft.com/en-us/azure/virtual-machines/image-version-snapshot-cli for details."
```

Comment:
Pls. add a reference to https://docs.microsoft.com/en-us/rest/api/compute/disks/get#hypervgeneration and https://docs.microsoft.com/en-us/rest/api/compute/snapshots/get#hypervgeneration